demos/common/cpp/pipelines/src/async_pipeline.cpp

```diff
@@ -15,50 +15,20 @@
 */
 
 #include "pipelines/async_pipeline.h"
-#include <cldnn/cldnn_config.hpp>
 #include <utils/common.hpp>
 #include <utils/slog.hpp>
 
 using namespace InferenceEngine;
 
-AsyncPipeline::AsyncPipeline(std::unique_ptr<ModelBase>&& modelInstance, const CnnConfig& cnnConfig, InferenceEngine::Core& engine) :
+AsyncPipeline::AsyncPipeline(std::unique_ptr<ModelBase>&& modelInstance, const CnnConfig& cnnConfig, InferenceEngine::Core& core) :
     model(std::move(modelInstance)) {
 
-    // --------------------------- 1. Load inference engine ------------------------------------------------
-    slog::info << "Loading Inference Engine" << slog::endl;
-
-    slog::info << "Device info: " << slog::endl;
-    slog::info<< printable(engine.GetVersions(cnnConfig.devices));
-
-    /** Load extensions for the plugin **/
-    if (!cnnConfig.cpuExtensionsPath.empty()) {
-        // CPU(MKLDNN) extensions are loaded as a shared library and passed as a pointer to base extension
-        IExtensionPtr extension_ptr = make_so_pointer<IExtension>(cnnConfig.cpuExtensionsPath.c_str());
-        engine.AddExtension(extension_ptr, "CPU");
-    }
-    if (!cnnConfig.clKernelsConfigPath.empty()) {
-        // clDNN Extensions are loaded from an .xml description and OpenCL kernel files
-        engine.SetConfig({ {PluginConfigParams::KEY_CONFIG_FILE, cnnConfig.clKernelsConfigPath} }, "GPU");
-    }
-
-    // --------------------------- 2. Read IR Generated by ModelOptimizer (.xml and .bin files) ------------
-    slog::info << "Loading network files" << slog::endl;
-    /** Read network model **/
-    InferenceEngine::CNNNetwork cnnNetwork = engine.ReadNetwork(model->getModelFileName());
-    /** Set batch size to 1 **/
-    slog::info << "Batch size is forced to 1." << slog::endl;
-    model->reshape(cnnNetwork);
-
-    // -------------------------- Reading all outputs names and customizing I/O blobs (in inherited classes)
-    model->prepareInputsOutputs(cnnNetwork);
-
-    // --------------------------- 4. Loading model to the device ------------------------------------------
-    slog::info << "Loading model to the device" << slog::endl;
-    execNetwork = engine.LoadNetwork(cnnNetwork, cnnConfig.devices, cnnConfig.execNetworkConfig);
+    execNetwork = model->loadExecutableNetwork(cnnConfig, core);
 
     // --------------------------- 5. Create infer requests ------------------------------------------------
```

Comment:
The number in comment isn't actual anymore. Please, fix or remove (best option for me).

Reply:
Done